pkg/common/nodeutil/node_test.go

```diff
@@ -4,13 +4,14 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/golang/protobuf/proto"
+	legacyProto "github.com/golang/protobuf/proto" // nolint: staticcheck // deprecated library needed until WithDetails can take v2
```

Comment:
We may create an issue to track this so we don't forget?

Reply:
I _think_ this will become a compilation error when grpc-go takes up the changes?